### PR TITLE
docs: add intial POC and Development Plan (closes #38)

### LINKED ISSUE #38: problem statement, poc & development plan
superseded by #35

### PR DESCRIPTION
**Pull request by aarnphm**
- **docs: add styling changes and development plans**
- **chore: add citations to csl**
- **fix: vercel installation**
- **chore: remove vercel config**

### CODE REVIEW

File: docs/content/DevelopmentPlan/DevelopmentPlan.md [hunk 1]
```diff
@@ -7,141 +7,333 @@ date: "2024-09-16"
...
-What coding standard will you adopt?
+### TS/JS
+
+[Google style guide](https://google.github.io/styleguide/) will be used.
+
+All related tools has been setup corespondingly.
+
+### Python
+
+a variation of Google style guide will be used.
```
Comment (nebrask) [inline docs/content/DevelopmentPlan/DevelopmentPlan.md:211]: Capitalize the first letter: "A variation of the Google style guide will be used."

File: docs/content/DevelopmentPlan/DevelopmentPlan.md [hunk 2]
```diff
@@ -7,141 +7,333 @@ date: "2024-09-16"
...
+> [!note] Meeting chair assignment
+> Follow the schedule for notetakers.
+
+Given that `tinymorph` will include two components (web-based editor and ML inference server), the following include
+a rough draft of each champion's responsibilities:
+
+- design work: The team will collaborate on general design works for interfaces, lead by Aaron Pham.
+- web-based editor: [Nebras Khan](https://github.com/nebrask).
+- ML inference server: [Waleed Malik](https://github.com/waleedmalik7)
+- ML attention intervention: Aaron Pham
```
Comment (waleedmalik7) [inline docs/content/DevelopmentPlan/DevelopmentPlan.md:78]: 1) we should probably link aarons github to this too ([Aaron Pham](https://github.com/https://github.com/aarnphm))

2) I believe Lucas will also be working alongside on the ML components

File: docs/content/DevelopmentPlan/DevelopmentPlan.md (hunk 1 above)
Comment (aarnphm) [reply docs/content/DevelopmentPlan/DevelopmentPlan.md]: updated

File: docs/content/DevelopmentPlan/DevelopmentPlan.md (hunk 2 above)
Comment (aarnphm) [reply docs/content/DevelopmentPlan/DevelopmentPlan.md]: updated

File: docs/content/DevelopmentPlan/DevelopmentPlan.md [hunk 3]
```diff
@@ -2,146 +2,337 @@
...
+- performance rendering with inlay within text buffers
+- Modal editing won't work.
+- e2e testing would be a hassle, given there are multiple components for tinymorph.
+- Cold start for inference server.
+- Mismanaging KV cache from vLLM internal block manager given that we are modifying attention activation directly.
+
+To adverse given risks, the following will be implemented:
+
+- Propose a low-fidelty prototype, followed by a design iteration for specific UX consideration.
+- Trained a small sets of [[glossary#low-rank adapters|LoRA]] as backup to merge with Llama 3.1 on inference server, given that LoRA has bene hardened by industry practice.
```
Comment (aarnphm) [inline docs/content/DevelopmentPlan/DevelopmentPlan.md:138]:
```suggestion
- Trained a small sets of [[glossary#low-rank adapters|LoRA]] as backup to merge with Llama 3.1 on inference server, given that LoRA has been hardened by industry practice.
```

**Review by lucas-lizhiwei — APPROVED** [review]

**Review by lucas-lizhiwei — APPROVED** [review]

**Pull request merged**